nodes/cma/nodes-cma-autoscaling-custom-debugging.adoc

```diff
@@ -0,0 +1,24 @@
+:_content-type: ASSEMBLY
+:context: nodes-cma-autoscaling-custom-debugging
+[id="nodes-cma-autoscaling-custom-debugging"]
+= Gathering debugging data
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+ifndef::openshift-origin[]
+When opening a support case, it is helpful to provide debugging information about your cluster to Red Hat Support.
+
+It is recommended that you provide the following information:
```

Comment:
@kcarmichael08 This is a **_very_** common phrase in our docs. Maybe need to purge them?

Reply:
Yeah, I always get it called out on my own PRs, but I realize it's really pervasive so I've been taking a phased approach to fixing it (I'll fix it when I see it in my docs, but have waited on any full-scale effort to fix just because of time restraints).